integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/BridgeContext.java

```diff
@@ -55,11 +53,8 @@ public String getEndPointPath() {
 
     public void setEndPoint(String endPoint) {
         this.endPoint = endPoint;
-        Matcher matcher = ENDPOINT_URL_REGEX.matcher(endPoint);
-        if (matcher.find()) {
-            endPointBaseUrl = matcher.group(1);
-            endPointPath = matcher.group(2);
-        }
+        endPointBaseUrl = Utils.getEndpointBaseUrl(endPoint).orElse(null);
```

Comment:
If endpoint base URL is not found, is it valid scenario? Can it happen in normal circumstances?
I would rather throw an error in such case, to fail fast.

Reply:
done